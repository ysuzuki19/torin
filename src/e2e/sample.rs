fn main() {
    println!("This is a placeholder for the main function.");
    // You can call other functions or implement logic here.
}

// torin DELETE BEGIN date=2025-10-01
fn e() {
    panic!("This function is not implemented yet");
}
// this line is ranged by torin BEGIN..END
// torin DELETE END

// this line is error by following line
// torin DELETE NEIGHBOR feature=debug
fn e2() {
    panic!("This function is not implemented yet");
}
// this line is error by above `torin`

// torin ERROR NEIGHBOR feature=debug
