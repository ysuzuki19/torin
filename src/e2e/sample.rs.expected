fn main() {
    println!("This is a placeholder for the main function.");
    // You can call other functions or implement logic here.
}


// this line is error by following line
// torin DELETE NEIGHBOR rule=debug
fn e2() {
    panic!("This function is not implemented yet");
}
// this line is error by above `torin`

// torin ERROR NEIGHBOR rule=debug
